client/src/app/index.tsx

```diff
@@ -1,10 +1,24 @@
-import React, { useEffect } from "react";
+import React, { useEffect, createContext, useState, useContext } from "react";
 import { View, Text } from "react-native";
 import { AuthStore } from "../services/store";
 import { useRootNavigationState, useRouter, useSegments } from "expo-router";
 
+export const SettingsContext = createContext({
```

Comment:
Let's keep the contexts within the Context folder for better compartmentalization and structure!

you can create a new file called settingContexts.tsx or something along those lines.

Reply:
Shouldn't we name the folder 'Settings' and have the context there to be consistent with how we created a context for the sockets?